extension/src/service/klarna-service.js

```diff
@@ -0,0 +1,86 @@
+const {
```

Comment:
I think it's a service, why not? But it can also be an utility or part of the klarna handler as well, I think it's kind of the same here. WDYT?

What do you mean that `createLineItems` should be encapsulated? Isn't it already encapsulated? It's the only method that is exposed in this service.

Reply:
It's not a service because the logic here is **only** part of the internals of the klarna payment handling (execute method) and it's just doing line items construction, you could rename it maybe sth like line-item-mapper, but it should not be part of the current scope when we got this as a new requirement we could extract those to not duplicate same logic but current scope does not need it.

In the project, we have only one service pattern which is web-component-service, and it's a service because this file is organized for reusing Adyen communication and it does not depend on any other execute method, even for that purpose we are not directly testing it because it could be changed easily.

Actually main reason is not naming, in detail here for that logics there is no reusability purpose, naming this as a service, exposing it and directly testing this instead of testing payment-handler: execute method is wrong. 
 write a test for your contracts not for your internal architectural models.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/294/commits/143cc441a51b4debc177f6f04f19c8a1a99b90c1